src/core/Table/filters/TableFilters.tsx

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { FilterButtonBarTranslation } from './FilterButtonBar';
+import { TextFilter } from './TextFilter/TextFilter';
+import { TableFilterProps } from './types';
+
+export const TableFilters = {
```

Comment:
Vykintas does not like this name, too. @mayank99 do you have a suggestion here? 😀

Reply:
So lets look at the usage:

```jsx
columns: [
  {
    id: 'name',
    Header: 'Name',
    Filter: TableFilters.TextFilter(),
    fieldType: 'text',
  },
  ...
]
```

I think it would make more sense to simply be able to do this:
```jsx
Filter: TextFilter // instead of TableFilters.TextFilter()
```
```jsx
Filter: TextFilter({ filter: "Filter", clear: "Clear" })
```

So why not just export it directly? The downside is that it will be too many exports from core index.

If we really want to export a single object, then lets not make it UpperCamelCase (which gives the impression that it's a component itself, not a map of components). So `tableFilters` or `predefinedFilters` or `filterVariants`.

Reply:
I would like filters to come from one place and IDE auto suggestions also helps to see what filter is available. I renamed it to lowercase `tableFilters`.